src/java.base/share/classes/sun/security/x509/CertAttrSet.java

```diff
@@ -63,7 +64,7 @@
      * @exception CertificateException on encoding or validity errors.
      * @exception IOException on other errors.
      */
-    void encode(OutputStream out)
+    void encode(DerOutputStream out)
```

Comment:
A bit of a nit, but on line 62, change `OutputStream` to `DerOutputStream` since it is referring to a type. Check the javadoc of all other classes for the same change.